Garrett-Iannuzzi/hot_like_sauce
# Add initial setup, component structure, and package install

## What does this PR do?
- Installs TS react router 
- Install SCSS
- Basic setup for `App`, `About`, `Product`, `Header`, and `Footer`

## How to test?
- Pull down feature branch
- Run `npm I`
- Browser should open and you will see each component 

## Issues:
- No issues open yet
- Project board will be set up tomorrow (4/9)

## Notes
- Added `box-sizing: border box` to the main `index.css` for layout purposes, will remove later.

@Garrett-Iannuzzi
